index.html

```diff
@@ -0,0 +1,272 @@
+      />
+    </div>
+
+    <nav class="breadcrumbs-navbar desktop-show">
+      <ul>
+        <li>
+          <a href="#">Basketball</a>
+        </li>
+        <li>
+          <a href="category_page.html">Football</a>
+        </li>
+        <li>
+          <a href="#">Tennis</a>
+        </li>
+        <li>
+          <a href="#">Swimming</a>
+        </li>
+        <li>
+          <a href="#">Chess</a>
+        </li>
+      </ul>
+    </nav>
+
+    <main>
+      <h1>Most Popular</h1>
+      <article class="carousel">
+        <h2>Customers' Choice</h2>
+        <div class="carousel-container">
+          <button type="button">&larr;</button>
+          <section class="product-card-carousel">
+            <div class="img-container">
+              <a href="#">
+                <img
+                  src="https://placeimg.com/640/480/tech"
+                  alt="product-photo"
+                />
+              </a>
+            </div>
+            <div class="data-container">
+              <h3 class="product-name-carousel">Product Name</h3>
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut
+                commodo, magna ac semper malesuada, orci magna varius magna, vel
+                pharetra massa justo sed erat. Proin nec imperdiet erat.
+              </p>
+              <p>Price</p>
+            </div>
+          </section>
+          <section class="product-card-carousel desktop-show">
+            <div class="img-container">
+              <a href="#">
+                <img
+                  src="https://placeimg.com/640/480/tech"
+                  alt="product-photo"
+                />
+              </a>
+            </div>
+            <div class="data-container">
+              <h3 class="product-name-carousel">Product Name</h3>
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut
+                commodo, magna ac semper malesuada, orci magna varius magna, vel
+                pharetra massa justo sed erat. Proin nec imperdiet erat.
+              </p>
+              <p>Price</p>
+            </div>
+          </section>
+          <section class="product-card-carousel desktop-show">
+            <div class="img-container">
+              <a href="#">
+                <img
+                  src="https://placeimg.com/640/480/tech"
+                  alt="product-photo"
+                />
+              </a>
+            </div>
+            <div class="data-container">
+              <h3 class="product-name-carousel">Product Name</h3>
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut
+                commodo, magna ac semper malesuada, orci magna varius magna, vel
+                pharetra massa justo sed erat. Proin nec imperdiet erat.
+              </p>
+              <p>Price</p>
+            </div>
+          </section>
+          <button type="button">&rarr;</button>
```

Comment:
Instead of html entities, I'd recommend using pseudoelements with unicode chars ;)